assets/src/edit-story/components/header/buttons.js

```diff
@@ -149,7 +155,10 @@ function Update() {
     default:
       text = __('Save draft', 'web-stories');
       return (
-        <Outline onClick={saveStory} isDisabled={isSaving || isUploading}>
+        <Outline
```

Comment:
I think you should be able to save a draft without a title.

Reply:
@spacedmonkey Only for `Save a draft`? How about `Schedule` and `Update`?

Reply:
Fixed!